airbyte-integrations/connectors/source-airtable/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_airtable ./source_airtable
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=4.0.0
```

Comment:
when adding a field or a stream, we should increment minor version, not patch

Reply:
Even if this field for internal use only?